http/headers/content-dpr.json

```diff
@@ -45,7 +45,7 @@
           "status": {
             "experimental": true,
             "standard_track": true,
```

Comment:
If I understand correctly, these aren't standard track either now, right? If there's no specification for them any more, it seems right to mark all of these as `false`.

Reply:
Agreed. Done.